excellence-fee/admin/admin_input.php

```diff
@@ -1,52 +1,22 @@
 <?php
+	global $wpdb;
 	// $insert_path=plugin_dir_path( __FILE__ ) . "insert_fee.php";
-	$insert_path=plugins_url( 'insert_fee.php', __FILE__ )
+	$insert_path=plugins_url( 'insert_fee.php', __FILE__ );
+
+	$current_value = $wpdb->get_results('select * from ' . $wpdb->prefix . 'posts where  post_name="excellence_fee"');
+
+	 foreach ($current_value as $key => $value) {
+    	$current_value=$value -> post_content;
+    	
+    }
 ?>
 
 <form method="post" action="<?php echo $insert_path; ?>" name="fee_form" class="fee_form">
 	<div class="input-form">
-		<div class="">Enter the Fee Amount: </div>
+		<div class="form_label"><label>Enter the Fee Amount: </label></div>
```

Comment:
@Shraddha1596  wrong way for writing string, correct everywhere need, all place have in wrong standard (no wp standard followed)